Code/Framework/AzCore/AzCore/DOM/DomBackendRegistry.cpp

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include <AzCore/DOM/DomBackendRegistry.h>
+
+#include <AzCore/Interface/Interface.h>
+
+namespace AZ::DOM
+{
+    BackendRegistry* BackendRegistry::s_instance = nullptr;
+
+    BackendRegistryInterface* BackendRegistry::Get()
+    {
+        return AZ::Interface<BackendRegistryInterface>::Get();
+    }
+
+    void BackendRegistry::Create()
+    {
+        AZ_Assert(!s_instance, "Attempted to register BackendRegistry when it's already registered");
+        s_instance = aznew BackendRegistry;
+        AZ::Interface<BackendRegistryInterface>::Register(s_instance);
```

Comment:
This is unsafe across multiple binaries as each will have a unique instance of `s_instance`. It's better to check if an interface is already registered, optionally cache the pointer, and only create if there's no interface yet.